docs/admin/resourcequota/index.md

```diff
@@ -84,45 +85,77 @@ You might want to set a pods quota on a namespace
 to avoid the case where a user creates many small pods and exhausts the cluster's
 supply of Pod IPs.
 
+## Quota Scopes
+
+Each quota can have an associated set of scopes.  A quota will only measure usage for a resource if it matches
+the intersection of enumerated scopes.
+
+When a scope is added to the quota, it limits the number of resources it supports to those that pertain to the scope.
+Resources specified on the quota outside of the allowed set results in a validation error.
+
+| Scope | Description |
+| ----- | ----------- |
+| Terminating | Match pods where `spec.activeDeadlineSeconds >= 0` |
+| NotTerminating | Match pods where `spec.activeDeadlineSeconds is nil` |
+| BestEffort | Match pods that have best effort quality of service. |
+| NotBestEffort | Match pods that do not have best effort quality of service. |
+
+A `BestEffort` scope restricts a quota to limit the following resources:
+* pods
+
+A `Terminating`, `NotTerminating`, and `NotBestEffort` scope restricts a quota to tracking the following resources:
```

Comment:
Do you think it'd sound better as "The `Terminating` ... scopes restrict a quota to tracking the following resources"? (same q for the previous para too.)